code/game/objects/items/plushes.dm

```diff
@@ -66,8 +107,143 @@
 		var/turf/T = get_turf(user)
 		log_game("[key_name(user)] added a grenade ([I.name]) to [src] at [COORD(T)].")
 		return
+	if(istype(I, /obj/item/toy/plush))
+		love(I, user)
+		return
 	return ..()
 
+/obj/item/toy/plush/proc/love(obj/item/toy/plush/Kisser, mob/living/user)	//~<3
+	var/chance = 100	//to steal a kiss, surely there's a 100% chance no-one would reject a plush such as I?
+	var/concern = 20	//perhaps something might cloud true love with doubt
+	var/loyalty = 30	//why should another get between us?
+	var/duty = 50		//conquering another's is what I live for
+
+	//we are not catholic
+	if (young == TRUE || Kisser.young == TRUE)
+		user.show_message("<span class='notice'>[src] plays tag with [Kisser].</span>", 1,
+			"<span class='notice'>They're happy.</span>", 0)
+	//never again
+	else if(Kisser in scorned)
+		//message, visible, alternate message, neither visible nor audible
+		user.show_message("<span class='notice'>[src] rejects the advances of [Kisser]!</span>", 1,
+			"<span class='notice'>That didn't feel like it worked.</span>", 0)
+	else if(src in Kisser.scorned)
+		user.show_message("<span class='notice'>[Kisser] realises who [src] is and turns away.</span>", 1,
+			"<span class='notice'>That didn't feel like it worked.</span>", 0)
+
+	//first comes love
+	else if(Kisser.lover != src && Kisser.partner != src)	//cannot be lovers or married
+		if(Kisser.lover)	//if the initiator has a lover
+			Kisser.lover.heartbreak(Kisser)	//the old lover can get over the kiss-and-run whilst the kisser has some fun
+			chance -= concern	//one heart already broken, what does another mean?
+		if(lover)	//if the recipient has a lover
+			chance -= loyalty	//mustn't... but those lips
+		if(partner)	//if the recipient has a partner
+			chance -= duty	//do we mate for life?
+
+		if(prob(chance))	//did we bag a date?
+			user.visible_message("<span class='notice'>[user] makes \the [Kisser] kiss \the [src]!</span>",
```

Comment:
`\the`se should be removed, all of `\the`m. `\the`y're unnecessary.
It's ra`\the`r strange that some of `\the`se don't have 'em, but most o`\the`rs do.

Reply:
_**Griff Murderbone** makes ratvar plushie kiss narsie plushie_
_**Griff Murderbone** makes the ratvar plushie kiss the narsie plushie_

I'll look at which ones should have `\the` parts removed or added. Would it be better to rename the toys? Having them all be `[something] plushie` takes up most word space.

Reply:
`\the [Kisser]` doesn't do anything different than `[Kisser]`. If Kisser has a proper name, then `\the` doesn't do anything and is omitted. If kisser has an improper name, `\the` is included. this is what you expect, but what you don't know is that just `[Kisser]` does the exact same thing even without `\the`, so we don't use `\the` anymore.

Reply:
`\the [Kisser]` doesn't do anything different than `[Kisser]`. If Kisser has a proper name, then `\the` doesn't do anything and is omitted. If kisser has an improper name, `\the` is included. This is what you expect, however, `[Kisser]` on its own does the same thing even without `\the`, so we don't use `\the` anymore.

Reply:
Yeah I found out that after testing. Sorry for not realising this was filtered for in the first place.

The `\the`s will be gone in the next commit, I'm just flailing at a feature breaking bug right now.